src/router.js

```diff
@@ -10,18 +10,16 @@ export default function (app) {
         }
       }
     },
-    hooks: {
-      onRender: function (model) {
-        return app.view[model.router.match]
-      }
-    },
-    subscriptions: [
-      function (_, actions) {
+    subscriptions: {
+      "render": function (model, view) {
```

Comment:
Most for consistency with the others. I suggest that we namespace this names, like: `core:render`, `route:enter`, and so on.